ecl_linear_algebra/include/ecl/linear_algebra/sophus/helpers.hpp

```diff
@@ -52,24 +52,24 @@ namespace Sophus {
 ** Interfaces
 *****************************************************************************/
 
-template<typename T>
-std::ostream & operator << ( std::ostream & out, const SE3Group<T> & se3 )
-{
-//  typename SE3Group<T>::Tangent tanget_vector = SE3Group<T>::log( se3 );
+//template<typename T>
```

Comment:
This is commented out because I wasn't having much success with sophus ... I'm sure a proper fix wouldn't comment it out.